packages/app-store/routing-forms/components/react-awesome-query-builder/widgets.tsx

```diff
@@ -129,7 +129,7 @@ function NumberWidget({ value, setValue, ...remainingProps }: TextLikeComponentP
       type="number"
       labelSrOnly={remainingProps.noLabel}
       containerClassName="w-full"
-      className="focus:border-brand-default bg-default dark:bg-muted  border-default disabled:bg-emphasis focus:ring-brand block w-full rounded-md text-sm disabled:hover:cursor-not-allowed"
+      className="bg-default dark:bg-muted border-default disabled:bg-emphasis block w-full rounded-md text-sm focus-within:border-neutral-300 focus-within:outline-none focus-within:ring-2 focus-within:ring-neutral-800 disabled:hover:cursor-not-allowed"
```

Comment:
@sean-brydon i think this should be using our color tokens no?

Reply:
Yes @gitstart-calcom  please convert this to our [tokens](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?type=design&t=RpKFVybDZL62nMnk-6)